dockstore-webservice/src/main/java/io/dockstore/webservice/core/LambdaEvent.java

```diff
@@ -168,11 +177,30 @@ public void setUser(User user) {
         this.user = user;
     }
 
+    public String getEntryName() {
+        return entryName;
+    }
+
+    public void setEntryName(String entryName) {
+        this.entryName = entryName;
+    }
+
+    public String getGroupId() {
+        return groupId;
+    }
+
+    public void setGroupId(String groupId) {
+        this.groupId = groupId;
+    }
+
     public enum LambdaEventType {
         PUSH,
         DELETE,
         INSTALL,
         PUBLISH
     }
 
+    public static String createGroupId() {
+        return UUID.randomUUID().toString();
```

Comment:
This is intentionally meaningless. Maybe this should be something like the `X-GitHub-Delivery` so we can debug and match webhook events and groups of logs. 
This may involve modifying the lambda and this table, so definitely feel free to break into a new ticket. 
https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads

Reply:
Ohh didn't realize that existed - the `X-GitHub-Delivery` is definitely more useful and could aid in implementing https://ucsc-cgl.atlassian.net/browse/SEAB-5604.

One thing I'm not sure of is how to assign the groupIds for existing lambda events if we use the GitHub delivery ID. Just randomly assign them an ID like I already do in this PR?

Reply:
I think random up to your fix is fine. We would probably only be able to debug a day or two backward anyway before https://ucsc-cgl.atlassian.net/browse/SEAB-3717

Reply:
Cool, I'll create a follow-up ticket to implement this (seems a little involved - tried adding a `deliveryId` parameter to the webhook endpoints and while the API code itself is straight forward, the change would likely require us to convert Swagger webhook tests to OpenAPI). 

In preparation for the follow up ticket, I'm going to rename the `groupId` column to `deliveryId` to reflect that this stores the `X-GitHub-Delivery` header value

Reply:
Hmm, the lambda event `id` or `dbcreateid` column might suffice if you're trying to see the order we processed the events in:
```
webservice_test=# select dbcreatedate, id from lambdaevent where groupid = 'f7e56fae-da6f-4764-9de9-554a1b0a6c33';
      dbcreatedate       | id 
-------------------------+----
 2023-07-27 09:27:12.442 |  1
 2023-07-27 09:27:12.457 |  2
 2023-07-27 09:27:13.182 |  3
 2023-07-27 09:27:15.191 |  4
 2023-07-27 09:27:18.049 |  5
 2023-07-27 09:27:19.21  |  6
 2023-07-27 09:27:21.215 |  7
 2023-07-27 09:27:22.541 |  8
 2023-07-27 09:27:24.571 |  9
(9 rows)
```

Reply:
That works, and the id is sent in the API response.

Reply:
> This is intentionally meaningless. Maybe this should be something like the X-GitHub-Delivery so we can debug and match webhook events and groups of logs.
This may involve modifying the lambda and this table, so definitely feel free to break into a new ticket.
https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads

I renamed the `groupId` column to `deliveryId` and created https://ucsc-cgl.atlassian.net/browse/SEAB-5781